addons/point_of_sale/static/src/app/navbar/cash_move_popup/cash_move_popup.js

```diff
@@ -82,4 +81,25 @@ export class CashMovePopup extends Component {
     _prepare_try_cash_in_out_payload(type, amount, reason, extras) {
         return [[this.pos.session.id], type, amount, reason, extras];
     }
+    toggleNegatifPositif() {
+        this.state.amount = -Number(this.state.amount);
+    }
+
+    async openDetailsPopup() {
+        const action = _t("Cash In/Out");
+        this.hardwareProxy.openCashbox(action);
+        this.dialog.add(MoneyDetailsPopup, {
+            moneyDetails: this.moneyDetails,
+            action: action,
+            getPayload: (payload) => {
+                const { total, moneyDetailsNotes, moneyDetails } = payload;
+                this.state.amount = this.env.utils.formatCurrency(total, false);
+                if (moneyDetailsNotes) {
+                    this.state.notes = moneyDetailsNotes;
+                }
+                this.moneyDetails = moneyDetails;
+            },
+            context: "Cash In/Out",
+        });
+    }
```

Comment:
This is the 3rd instance of this method, can you check if it's worth it to extract a common method for this?